layers/+tools/xclipboard/funcs.el

```diff
@@ -0,0 +1,96 @@
+;;; funcs.el --- xclipboard layer functions file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Google Inc. & Contributors
+;;
+;; Author: Charles Weill <weill@google.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defun xclipboard/get-display ()
```

Comment:
We usually put all the functions in `funcs.el` under the `spacemacs/` prefix.
For instance this one should be renamed `spacemacs/xclipboard-get-display`.